Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -687,18 +685,6 @@ public IEnumerable<T> GetExports<T>(bool manageLifetime = true)
             return parts;
         }
 
-        // TODO: @bond
-        /*
-        private void SetBaseExceptionMessage()
-        {
-            var builder = GetBaseExceptionMessage(ApplicationPaths);
-
-            builder.Insert(0, string.Format("Version: {0}{1}", ApplicationVersion, Environment.NewLine));
```

Comment:
What would be means of knowing the version using logs only? This bit might be useful.

Reply:
The version is getting logged somewhere else already: https://github.com/jellyfin/jellyfin/blob/dev/Jellyfin.Server/Program.cs#L57